### PR TITLE
Image compression workflow: ignore `kirby` folder

### PR DESCRIPTION
**Pull request by distantnative**

### CODE REVIEW

**Comment by lukasbestle** [comment]
I don't think this will work. It will only control when the workflow will be executed, but if it executes because of changed images in the `content` dir, it will still compress images everywhere.

See https://github.com/calibreapp/image-actions#ignore-paths.

**Comment by distantnative** [comment]
Added a restriction on the step as well.

**Comment by distantnative** [comment]
Have you found if there's a good way to test GitHub Actions locally?

File: .github/workflows/images.yml [hunk 1]
```diff
@@ -22,7 +23,7 @@ on:
       - '**.jpeg'
       - '**.png'
       - '**.webp'
-  workflow_dispatch:
```
Comment (lukasbestle) [inline .github/workflows/images.yml:25]: If this was removed on purpose, please also update the comment at the beginning of the file. That also still mentions that the workflow is run on schedule.

**Comment by lukasbestle** [comment]
> Have you found if there's a good way to test GitHub Actions locally?

There is https://github.com/nektos/act, but I haven't tried that as I don't want to set up Docker on my Mac. It also doesn't seem to replicate the GitHub Actions feature set 100 %. So I have used test branches directly on GitHub so far.

**Review by lukasbestle — APPROVED** [review]

**Pull request merged**